### PR TITLE
small style changes to text

### LINKED ISSUE #432: Joni Mitchel Missing Paintings
Some of the paintings on Joni Mitchel's page are missing. These need to be found, and re-added. Worst case scenario, check NAC for the images.

### PR DESCRIPTION
**Pull request by AndrewJMD**
the links now look better
all text is white on the Album and Painting pages

fixes #432

### CODE REVIEW

File: www/artists/jmitchell2/album.html [hunk 1]
```diff
@@ -11,36 +11,36 @@
     </head>
     <body>
         <h1>
-            Joni's Album Art &#9829;
+            Joni's Album Art &hearts;
         </h1>
-        <div id="nav">
-            <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
+        <div id="nav" class="nav">
```
Comment (gemohandy) [inline www/artists/jmitchell2/album.html]: Why does this navigation div have a nav class, and none of the others? Particularly when the nav class isn't defined in style.css, and has no effect on anything. (The same can be said about the nav id, but at least it's consistent.)

File: www/artists/jmitchell2/index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     </head>
     <body>
         <h1>
-            Joni Mitchell &#9829;
+            Joni Mitchell &hearts;
         </h1>
         <div id="nav">
             <a href="index.html">Home</a> <a href="album.html">Album Artworks</a> <a href="paintings.html">Paintings</a> <a href="discography.html">Discography</a> <a href="/">Saskatchewan Artists</a> <a href="http://www.jonimitchell.com">Joni Mitchell's Site</a>
```
Comment (gemohandy) [inline www/artists/jmitchell2/index.html]: Why doesn't this navbar get lines between the links? The others do.

File: www/artists/jmitchell2/style2.css [hunk 3]
```diff
@@ -17,10 +17,13 @@ h1{
...
+.title{
+	text-align: center;
+	color: white;
 }
 
-a:hover{
-	color: pink; 
-}
\ No newline at end of file
+a, a:visited{
```
Comment (gemohandy) [inline www/artists/jmitchell2/style2.css]: As a general rule, we prefer to preserve the original coloration of the sites. Would it be possible to make the links yellow normally, and pick on hover, as it originally was? 
Also, colouring the links white with no class restriction is probably a bad idea - there are links on index.html, which has a white background. Are the links even visible there with this change?

Comment (AndrewJMD) [reply www/artists/jmitchell2/style2.css]: there are 2 style sheets and the one i edited was for the artwork and paintings only, i can revert it back to being yellow and pink for those two. I decided to only modify those 2 pages as they are the only ones where text was hard to read and preserving the other pages is important

**Pull request merged**